tests/unit_tests/test_data_dose.py

```diff
@@ -0,0 +1,29 @@
+from openmc.data import dose_coefficients
+from pytest import approx, raises
+
+
+def test_dose_coefficients():
+    # Spot checks on values from ICRP tables
+    energy, dose = dose_coefficients('photon', 'AP')
+    assert energy[0] == approx(0.01e-6)
+    assert dose[0] == approx(0.0685)
+    assert energy[-1] == approx(10e-3)
+    assert dose[-1] == approx(90.4)  # updated in corrigendum
```

Comment:
There is a corrigendum that is separate from the original report. The supplemental data includes the corrected values found in the corrigendum. I can add a comment that states this explicitly- just wasn't sure initially if it was worth saying that we are using the right values, not the wrong values. Let me know what you prefer, or feel free to make a suggested change.

Reply:
Actually, I guess since the docstring includes a link to the original report, I will go ahead and add a comment that the values from the corrigendum are used.